src/pages/getstarted/GetStarted.js

```diff
@@ -7,6 +7,8 @@ import { colors, media } from '../../components/theme';
 import DescriptionList from './DescriptionList';
 import ResourcesList from './ResourcesList';
 
+import LearningList from './LearningList';
```

Comment:
remove import for `DescriptionList`. if it is no longer used. delete it.
Also probably the json https://github.com/devinea/mvx19-cms/blob/master/src/data/getstarted.json#L2-L56

Reply:
Import for import for `DescriptionList` removed.

In relation to https://github.com/devinea/mvx19-cms/blob/master/src/data/getstarted.json#L2-L56, the below error occurs when removed. As the DescriptionList.js and Description.js components are no longer in use, shall we delete them?

`error GraphQL Error Unknown field `read` on type `[data_4]`

  file: /Users/i505380/Projects/mvx19-cms/src/pages/getstarted/DescriptionList.js

   9 |                 title
  10 |                 data {
  11 |                   title
  12 |                   image {
  13 |                     src
  14 |                     width
  15 |                     height
  16 |                   }
  17 |                   description
  18 |                   url
> 19 |                   read
     |                   ^
  20 |                 }
  21 |               }
  22 |             }
  23 |           }
  24 |         }
  25 |       }
  26 |`

Reply:
I think they should be removed. @eouin ok?

Reply:
Yep if they are not in use anywhere else then hey should be removed

Reply:
Thanks, working on it.

Reply:
@devinea @eouin Updates and deletions made and pushed to branch.